babel/src/main/codegen/config.fmpp

```diff
@@ -985,12 +985,19 @@ data: {
     ]
 
     # List of methods for parsing extensions to "CREATE [OR REPLACE]" calls.
-    # Each must accept arguments "(SqlParserPos pos, boolean replace)".
+    # Each must accept arguments:
+    # "(SqlParserPos pos, SqlCreate.SqlCreateSpecifier createSpecifier)".
```

Comment:
nit: Add "org.apache.calcite.sql.SqlCreate.SqlCreateSpecifier" in the imports at line 24, and use the Enum name SqlCreateSpecifier rather than SqlCreate.SqlCreateSpecifier in config.fmpp, parserImpls.ftl, and in code.

Reply:
Correct me if there is a particular reason that drive you to use SqlCreate.SqlCreateSpecifier rather than SqlCreateSepcifier.

Reply:
Good point, will do.

Reply:
I think if it's changed here, it might be good to also change it throughout the Java files because some of them reference SqlCreate.SqlCreateSpecifier and other Java files just reference SqlCreateSpecifier